src/main/java/uk/gov/hmcts/reform/civil/model/CaseDataParent.java

```diff
@@ -747,8 +747,10 @@ public boolean hasClaimantAgreedToFreeMediation() {
     }
 
     @JsonIgnore
-    public Optional<TranslatedDocument> getTranslatedDocument() {
-        return Optional.ofNullable(getCaseDataLiP()).map(CaseDataLiP::getTranslatedDocument);
+    public List<Element<TranslatedDocument>> getTranslatedDocument() {
+        return Optional.ofNullable(getCaseDataLiP())
+            .map(CaseDataLiP::getTranslatedDocument)
+            .orElse(null);
```

Comment:
do we want this to return null or n empty list? null is not very good to return because the other side would have to deal with it. also should be getTranslatedDocuments as it now returns a list

Reply:
fixed